rust/tvm-sys/src/byte_array.rs

```diff
@@ -0,0 +1,64 @@
+use std::os::raw::c_char;
+
+use crate::ffi::TVMByteArray;
+
+/// A struct holding TVM byte-array.
```

Comment:
```suggestion
/// A struct holding TVM byte-array.
```
is it really holding? looks pretty direct to me. what interface are you going for? maybe do a newtype wrapper? dunno

Reply:
This comment also comes from the original code.